toolkit/tools/scheduler/scheduler.go

```diff
@@ -115,6 +117,11 @@ func main() {
 		logger.Log.Fatalf("Value in --build-attempts must be greater than zero. Found %d.", *buildAttempts)
 	}
 
+	if *maxCascadingRebuilds < 0 {
+		logger.Log.Infof("--max-cascading-rebuilds (%d) is negative, setting to unbounded cascading rebuilds.", *maxCascadingRebuilds)
```

Comment:
Since the default behaviour is unbounded, printing this on info level might be confusing for the users. Especially since they don't necessarily are aware of any argument names passed to single Go tools but rely on the `make` arguments instead. I'd suggest moving this to debug level.

```suggestion
		logger.Log.Debugf("--max-cascading-rebuilds (%d) is negative, setting to unbounded cascading rebuilds.", *maxCascadingRebuilds)
```

Reply:
I see now that we do very similar logging inside `NewGraphBuildState`. I'm curious if we need this `if` at all in that case.

Reply:
Removed in favor of the internal package version.